site/docs/hosting/cyclic.md

```diff
@@ -0,0 +1,157 @@
+---
+prev: false
+next: false
+---
+
+# Hosting: Cyclic
+
+This guide tells you about the ways you can host your grammY bots on [Cyclic](https://cyclic.sh/), using Node.js.
+
+## Prerequisites
+
+To follow along, you need to have a [Github](https://github.com/) and [Cyclic](https://cyclic.sh/) account.
+First, install some dependencies:
+
+```sh
+# Create a project directory.
+mkdir grammy-bot
+cd grammy-bot
+npm init --y
+
+# Install main dependencies.
+npm install grammy express dotenv
+
+# Install development dependencies.
+npm install -D typescript ts-node nodemon @types/express @types/node
+
+# Create TypeScript config.
+npx tsc --init
+```
+
+We will store our TypeScript files inside a folder `src`, and our compiled files in a folder `dist`.
+Create the folders in the project's root directory.
+Then, inside folder `src`, create a new file named `bot.ts`.
+Our folder structure should now look like this:
+
+```asciiart:no-line-numbers
+.
+├── node_modules/
+├── dist/
+├── src/
+│   └── bot.ts
+├── package.json
+├── package-lock.json
+└── tsconfig.json
+```
+
+After that, open `tsconfig.json` and change it to use this configuration:
+
+```json
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "./src",
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "strict": true
+  }
+}
+```
+
+And then we have to add `start`, `build` and `dev` scripts to our `package.json`.
+Our `package.json` should now be similar to this:
+
+```json{6}
+{
+  "name": "grammy-bot",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index.js",
+    "dev": "nodemon src/index.ts"
+  },
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.18.2",
+    "grammy": "^1.17.2"
+  },
+  "devDependencies": {
```

Comment:
Ah hang on, we need to list nodemon here

Reply:
https://github.com/grammyjs/website/pull/888/commits/ec509cdb5f562266ead2ef76688a417b43a2d6b1